apps/schools/domains/common/utils/searchText.ts

```diff
@@ -0,0 +1,7 @@
+export function getSearchText(text: string) {
+    return text.split(':')[0]
+}
+
+export function createSearchTextForRequest(text: string, searchColumns: string[]) {
+    return `${text}:[${searchColumns.map((e: string) => e)}]`
```

Comment:
is there , in separator role?

Reply:
The separator in this case will be a comma, by default the map method separates them with commas